<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="Zafar" failures="1" tests="1" name="Default test(failed)(failed)(failed)" time="2.615" errors="0" timestamp="2023-03-01T00:35:50 IST">
  <testcase classname="TestNgPrectice.BasePracticeClass" name="test4" time="0.01">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at TestNgPrectice.BasePracticeClass.test4(BasePracticeClass.java:97)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test4 -->
  <testcase classname="TestNgPrectice.BasePracticeClass" name="test3" time="0.0">
    <ignored/>
  </testcase> <!-- test3 -->
  <testcase classname="TestNgPrectice.BasePracticeClass" name="test2" time="0.0">
    <ignored/>
  </testcase> <!-- test2 -->
  <testcase classname="TestNgPrectice.BasePracticeClass" name="test1" time="0.0">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed)(failed)(failed) -->
